docs/source/models.md

```diff
@@ -121,16 +126,17 @@ Example of a condition with two _new owners_:
     - `weight`: integer weight for each subfulfillment's contribution to the threshold
```

Comment:
I have a feeling we are replicating the docs of crypto-conditions/ILP spec

These (weight, threshold) are attributes of a ThresholdCondition